tools/cockpit.spec

```diff
@@ -142,13 +142,15 @@ The Cockpit Web Service listens on the network, and authenticates users.
 %setup -q
 
 %build
+exec 2>&1
 %if %{defined gitcommit}
 env NOCONFIGURE=1 ./autogen.sh
 %endif
-%configure --disable-static --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding} --with-selinux-config-type=etc_t
+%configure --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding} --with-selinux-config-type=etc_t
```

Comment:
Because it's not supported, and just makes configure.ac spew noise. The commit this is in is one that removes noise.